src/vs/editor/browser/config/charWidthReader.ts

```diff
@@ -56,7 +56,7 @@ class DomCharWidthReader {
 		this._readFromDomElements();
 
 		// Remove the container from the DOM
-		targetWindow.document.body.removeChild(this._container!);
+		this._container!.remove();
```

Comment:
```suggestion
		this._container?.remove();
```

Reply:
I deliberately chose to keep the same functionality here, but I agree your suggestion is an improvement.